generators/app/generator.js

```diff
@@ -158,6 +158,11 @@ module.exports = class extends Generator {
       name: 'lang',
       message: 'Do you need video.js language file infrastructure for internationalized strings?',
       default: defaults.lang
+    }, {
+      type: 'confirm',
+      name: 'library',
+      message: 'Do you want to every file in ./src available for consumption in other packages?',
```

Comment:
Had trouble understanding this sentence initially... maybe `Do you want every file in ./src to be available for use in other packages?"` or something like that?

Reply:
improved the wording and added mention of `./es` and `./cjs` so that it is even more clear.